plugins/cliconf/ios.py

```diff
@@ -198,6 +198,16 @@ def edit_config(
 
                 cmd = line["command"]
                 if cmd != "end" and cmd[0] != "!":
+                    if (
+                        self.send_command(**line)
+                        and len(self.send_command(**line).split(" ")) > 1
+                        or "%" in self.send_command(**line).split(" ")
```

Comment:
This behaviour change seems backwards-incompatible for `ios_config`/`cli_config` modules as warning from the device is now treated as an error. Can this scenario be handled in the respective resource modules?